### PR TITLE
make navbar responsive

### PR DESCRIPTION
**Pull request by olaoluwa-98**
Make navbar responsive

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -19,21 +19,32 @@
...
-    <ul class="nav navbar-nav">
+    <div class="collapse navbar-collapse" id="myNavbar">
+      <ul class="nav navbar-nav">
       <li {% block nav_home %}class="active"{% endblock %}><a href="{% url 'home:home' %}">Home</a></li>
       <li {% block nav_resources %}{% endblock %}><a href="{% url 'home:resources' %}">Resources</a></li>
     </ul>
+    </div>
   </div>
 </nav>
+
```
Comment (divyam3897) [inline templates/base.html]: Remove these empty lines.

File: templates/base.html [hunk 2]
```diff
@@ -19,19 +19,26 @@
     <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css" rel="stylesheet"/>
 </head>
     <!-- Latest compiled and minified JavaScript -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
-
```
Comment (sagar-kalra) [inline templates/base.html]: Remove this change.

Comment (olaoluwa-98) [reply templates/base.html]: This change is needed for the responsiveness, bootstrap needs jQuery

Comment (sagar-kalra) [reply templates/base.html]: Sorry, I understood it wrong. My mistake.
Just resolve conflics and squash your commits

File: templates/base.html [hunk 3]
```diff
@@ -19,38 +19,49 @@
...
-  <body  {% block body_style %} {% endblock %}>
 <!--CSS Loader -->
 <div class="loader-wrapper">
   <div class="loader">
-  	<div class="loader-rect1"></div>
-  	<div class="loader-rect2"></div>
-  	<div class="loader-rect3"></div>
-  	<div class="loader-rect4"></div>
-  	<div class="loader-rect5"></div>
+    <div class="loader-rect1"></div>
```
Comment (divyam3897) [inline templates/base.html]: Why did you do these changes?

Comment (olaoluwa-98) [reply templates/base.html]: It was coming from the merge

Comment (divyam3897) [reply templates/base.html]: do a reset --hard for the merge, rebase from master.

**Comment by olaoluwa-98** [comment]
I have an issue, while trying to squash, there's been some commits from other people. How can I do the squashing without it affecting the other people's commits

**Comment by divyam3897** [comment]
you had to a rebase, it places your changes above the changes of others.  see https://stackoverflow.com/questions/7297379/how-do-you-rebase-the-current-branchs-changes-on-top-of-changes-being-merged-in

**Comment by olaoluwa-98** [comment]
I think I have fixed the issues now

File: templates/base.html [hunk 4]
```diff
@@ -17,40 +17,52 @@
...
+  
 <!--CSS Loader -->
 <div class="loader-wrapper">
   <div class="loader">
-  	<div class="loader-rect1"></div>
-  	<div class="loader-rect2"></div>
-  	<div class="loader-rect3"></div>
-  	<div class="loader-rect4"></div>
-  	<div class="loader-rect5"></div>
+      <div class="loader-rect1"></div>
```
Comment (divyam3897) [inline templates/base.html]: I can still see these changes which should not be there :(

Comment (olaoluwa-98) [reply templates/base.html]: this was not made by me, it was someone else's commit

Comment (divyam3897) [reply templates/base.html]: that's what I am saying you should not have any of them!!.

**Comment by olaoluwa-98** [comment]
I have made neccessary changes

File: templates/base.html [hunk 5]
```diff
@@ -17,40 +17,33 @@
...
-<div class="loader-wrapper">
-  <div class="loader">
-  	<div class="loader-rect1"></div>
-  	<div class="loader-rect2"></div>
-  	<div class="loader-rect3"></div>
-  	<div class="loader-rect4"></div>
-  	<div class="loader-rect5"></div>
-  </div>
-</div>
+
```
Comment (divyam3897) [inline templates/base.html]: Remove this change

**Comment by olaoluwa-98** [comment]
I reverted to the commit before then, then made the changes. I think this solves it.

**Comment by olaoluwa-98** [comment]
I've added the jQuery dependency that bootstrap js needs

**Comment by divyam3897** [comment]
It has been merged, thank you :)

**Pull request merged**